include/git2/repository.h

```diff
@@ -193,6 +193,9 @@ typedef struct {
 	/**
 	 * A GIT_PATH_LIST_SEPARATOR delimited list of path prefixes at which
 	 * the search for a containing repository should terminate.
+	 *
+	 * FIXME: this depends on the value of GIT_PATH_LIST_SEPARATOR, which is
+	 * platform-specific. Make that a git_strarray ?
```

Comment:
I think it's perfectly fine to fix this API while we're at it. The old function could simply split the string it got into the string array